python/ray/tune/trial_runner.py

```diff
@@ -385,6 +391,15 @@ def has_resources(self, resources):
         """Returns whether this runner has at least the specified resources."""
         return self.trial_executor.has_resources(resources)
 
+    def _stop_experiment_if_needed(self):
+        """Stops all trials if the user condition is satisfied."""
+        if not self._stopper:
+            return
+
+        if self._stopper.stop_all():
+            [self.trial_executor.stop_trial(t) for t in self._trials]
+            logger.info("trial_runner: All trials stopped.")
```

Comment:
nit: doesn't `trial_runner.py` get printed by the logger anyways?
```suggestion
            logger.info("All trials stopped due to meeting stopper `stop_all` condition.")
```